force-app/main/default/lwc/geTemplateBuilder/geTemplateBuilder.js

```diff
@@ -1038,6 +1068,18 @@ export default class geTemplateBuilder extends NavigationMixin(LightningElement)
         this.formSections = formSections;
 
         this.removeOptionFromBatchTableColumn(fieldMappingDeveloperName);
+
+        this.announceFieldRemovedFromSection(section, field);
+    }
+
+    announceFieldRemovedFromSection(section, field) {
+        const formSectionIndex = findIndexByProperty(this.formSections, ID, section.id);
+        const screenReaderMessage =
+            section.label !== '' ?
```

Comment:
Is this label always going to be empty string when unnamed, or is there a chance of null/undefined here?

Reply:
The label property _should_ always exist on the section object, but yeah better to guard against the possibility of null/undefined. Thanks for the catch!